packages/block-editor/src/store/index.js

```diff
@@ -34,14 +34,14 @@ export const store = createReduxStore( STORE_NAME, {
 	...storeConfig,
 	persist: [ 'preferences' ],
 } );
+unlock( store ).registerPrivateActions( privateActions );
+unlock( store ).registerPrivateSelectors( privateSelectors );
 
 // We will be able to use the `register` function once we switch
 // the "preferences" persistence to use the new preferences package.
 const registeredStore = registerStore( STORE_NAME, {
 	...storeConfig,
 	persist: [ 'preferences' ],
 } );
-
-unlock( registeredStore ).registerPrivateActions( {
-	__experimentalUpdateSettings,
-} );
+unlock( registeredStore ).registerPrivateActions( privateActions );
+unlock( registeredStore ).registerPrivateSelectors( privateSelectors );
```

Comment:
Guessing I need to register private action/selectors on both the `store` and `registeredStore`?

Reply:
Locking only the `registeredStore` should be enough.. It didn't work for you if you did that? 🤔